src/main/parser.ts

```diff
@@ -389,6 +392,32 @@ export function createParser(tokens: Array<Token>, report: ErrorReporter = noopR
     return null
   }
 
+  // Annotations → '(' (Identifier '=' StringConstant ListSeparator?)* ')'
+  function parseAnnotations(): Array<Annotation> | null {
+    if (check(SyntaxType.LeftParenToken)) {
+      const annotations: Array<Annotation> = []
+      advance()
+      while (!check(SyntaxType.RightParenToken)) {
+          const nameToken: Token = consume(SyntaxType.Identifier)
+          consume(SyntaxType.EqualToken)
+          const next: Token = advance()
+          const value: StringConstant = parseStringValue(next)
```

Comment:
I am curious what we gain from adding StringConstant instead of using a StringLiteral as the value of an Annotation.

Reply:
You're right, there's no need to add a new StringConstant type. Wasn't very familiar with the code-base, so thought we'd want to return constant types like Floats and Doubles do, but given that e.g. include definitions just use StringLiterals, I'll remove the StringConstant.